### PR TITLE
docs: standardize text formatting

### PR DESCRIPTION
**Pull request by Meschreiber**
This PR applies some recent updates to the [docs style guide](https://docs.google.com/document/d/1jZhBYPgoZsxe5UEMD3oaGHISoxo6DFR7oJxNij0i6X8/edit#heading=h.8f9rtvqsv9):
- Reserve bold text for UI elements
- Reserve italics for a new/key concept
- Use standardized admonition components
- Minimize use of exclamation points
- Refer to "GraphOS Studio" rather than "Apollo Studio"

etc.

### CODE REVIEW

File: docs/source/managed-federation/setup.mdx [hunk 1]
```diff
@@ -2,11 +2,13 @@
 title: Setting up managed federation
 ---
 
-import ObtainGraphApiKey from '../../shared/obtain-graph-api-key.mdx';
```
Comment (Meschreiber) [inline docs/source/managed-federation/setup.mdx:5]: There is now a global `ObtainGraphAPIKey` that was kept more up-to-date.

File: docs/source/quickstart/local-subgraphs.mdx [hunk 2]
```diff
@@ -66,11 +74,11 @@ const server = new ApolloServer({
 
 (As shown, you now pass your `typeDefs` and `resolvers` to `buildSubgraphSchema`.)
 
-And that's it! You can now perform all of the same subgraph setup on your _own_ service (schema publishing, introspection, etc.) that you did with the Apollo-hosted services we used in the previous parts. Refer to those parts for guidance.
+And that's it! You can now perform all of the same subgraph setup on your own service (schema publishing, introspection, etc.) that you did with the Apollo-hosted services we used in the previous parts. Refer to those parts for guidance.
```
Comment (shorgi) [inline docs/source/quickstart/local-subgraphs.mdx:77]: I like the exclamation point for the congratulatory sentence, but I don't like `And that's it`. Maybe just `Congratulations`? 

Or should we avoid congratulatory sentences?

```suggestion
Congratulations! You can now perform all of the same subgraph setup on your own service (schema publishing, introspection, etc.) that you did with the Apollo-hosted services we used in the previous parts. Refer to those parts for guidance.
```

**Review by shorgi — APPROVED** [review]
Spot reviewed, nice job getting all these pages updated!

File: docs/source/entities-advanced.mdx [hunk 3]
```diff
@@ -709,13 +717,15 @@ type Product @key(fields: "id") {
...
+
+<MinVersion version="2.1.2">
 
 ### Using `@requires` with fields that take arguments
 
-> This functionality was introduced in Federation `v2.1.2`.
+</MinVersion>
 
-The `@requires` directive can include fields that take arguments, like so:
+Starting in Federation `v2.1.2`, the `@requires` directive can include fields that take arguments, like so:
```
[the remark this answers is not in the document]
Comment (Meschreiber) [reply docs/source/entities-advanced.mdx:728]: Added here: https://docs.google.com/document/d/1jZhBYPgoZsxe5UEMD3oaGHISoxo6DFR7oJxNij0i6X8/edit#heading=h.fc4s9rz8lxzk

File: docs/source/quickstart/local-subgraphs.mdx (hunk 2 above)
Comment (Meschreiber) [reply docs/source/quickstart/local-subgraphs.mdx]: I think some congratulations are alright, particularly at the end of lengthy quickstarts.

File: docs/source/migrating-from-stitching.md [hunk 4]
```diff
@@ -6,13 +6,17 @@ description: How to move your services to Apollo Federation
...
+<Tip>
+
+For a real-world example of an organization benefiting from this migration, see [this blog post](https://www.apollographql.com/blog/announcement/expedia-improved-performance-by-moving-from-schema-stitching-to-apollo-federation/).
+
+</Tip>
 
 ## Summary of steps
 
-This guide describes a set of steps for migrating architecture **incrementally** from stitching to federation. In order to do so, we rely on the fact that changes necessary for federation are completely backwards compatible. In other words, services that implement a part of your graph (**subgraphs**) can be used in both your stitching gateway and your Apollo gateway.
+This guide describes a set of steps for migrating architecture incrementally from stitching to federation. In order to do so, we rely on the fact that changes necessary for federation are completely backwards compatible. In other words, services that implement a part of your graph, also known as _subgraphs_, can be used in both your stitching gateway and your Apollo gateway.
```
[the remark this answers is not in the document]
Comment (Meschreiber) [reply docs/source/migrating-from-stitching.md]: Vale update here: https://github.com/apollographql/docs/pull/796

**Pull request merged**